tests/resources/impala_test_tables.sql

```diff
@@ -111,3 +111,52 @@ INSERT INTO `pso_data_validator`.`test_generate_partitions`
 ,('TRI001',1,1234,2.1),('TRI001',1,5678,3.5),('TRI001',1,9012,2.3)
 ,('TRI001',2,1234,3.5),('TRI001',2,5678,2.6),('TRI001',2,9012,3.5)
 ,('TRI001',3,1234,2.7),('TRI001',3,5678,3.5),('TRI001',3,9012,2.8);
+
+ /* Following table used for validating generating table partitions  version 2*/
+drop table if exists `pso_data_validator`.`test_generate_partitions_v2` ;
+CREATE TABLE `pso_data_validator`.`test_generate_partitions_v2` (
+        course_id STRING,
+        quarter_id INTEGER,
+        recd_timestamp TIMESTAMP,
+        registration_date DATE,
+        approved Boolean,
+        grade DECIMAL(5,2))
+STORED AS PARQUET
+TBLPROPERTIES ('comment'= 'Table for testing generate table partitions,
```

Comment:
Nit: this table comment is quite different to text used for BigQuery, Hive, SQL Server.

As I've read further perhaps this one is correct and the other three need updating?